examples/todo_list_butterfly/web/main.dart

```diff
@@ -0,0 +1,174 @@
+// Copyright (c) 2016, Alexei Eleusis Díaz Vera. All rights reserved. Use of this source code
+
+// is governed by a BSD-style license that can be found in the LICENSE file.
+
+import 'dart:html' as html;
+
+import 'package:butterfly/butterfly.dart';
+import 'package:greencat/greencat.dart';
+import 'package:logging/logging.dart';
+import 'package:todo_list/todo_list.dart';
+import 'package:tuple/tuple.dart';
+
+void main() {
+  runApp(new TodoApp(), html.document.querySelector('#output'));
+}
+
+/// App to manage todos.
+class TodoApp extends StatefulWidget {
+  @override
+  State<StatefulWidget> createState() {
+    return new _TodoAppState();
+  }
+}
+
+class _TodoAppState extends State<TodoApp> {
+  /// Store built with the combine ...
+  final Store<Tuple2<Iterable<Todo>, VisibilityFilter>, TodoAction>
+  combinedStore = new Store.createStore(todoCombinedApp);
+
+  _TodoAppState() {
+    combinedStore.dispatch(addTodo('take care of the green caterpillar'));
+    combinedStore.dispatch(addTodo('wait until it becomes a butterfly :)'));
```

Comment:
@yjbanov I also noticed the fact that he is sharing the todo list when i played with this code Its pretty cool.  I am really interested in the way butterfly works. The syntax is very flutter like and from knowing flutter when i looked at your code example it was immediately apparent how i can program using butterfly.  Its a huge payoff IMHO. I really hope this takes off. Also i noticed how easy it was to make MD widgets, even though you only have Button for now.  Congrats !